.travis.yml

```diff
@@ -4,7 +4,11 @@ language: minimal
 
 install:
   - sudo pip install mkdocs-material
+  - gem install html-proofer
 
 script:
   - git remote set-url origin https://${GITHUB_TOKEN}@github.com/bluerobotics/ping-viewer
+  - mkdocs build
+  - htmlproofer --empty-alt-ignore --file-ignore "./site/404.html" --url-ignore "https://fonts.gstatic.com" || exit 1
```

Comment:
I believe that will not finish the pipeline here, you need to call `travis_terminate`.